.github/workflows/build.yml

```diff
@@ -137,7 +164,7 @@ jobs:
             echo "::error::Unknown architecture: '${{ matrix.arch }}'"
             exit 1
           fi
-          NIX_FAST_BUILD_GITREF="1775c732"
+          NIX_FAST_BUILD_GITREF="906af17f"
```

Comment:
Usually the one from nixpkgs should be fine as well and saves some downloads

```
nix run --inputs-from .# nixpkgs#nix-fast-build -- args...
```

Reply:
Fixed in https://github.com/tiiuae/ghaf/pull/949/commits/230acca0db0dbea89fceec3eeb8011bd75fcdaa4